### ISSUE TITLE
Expose memory usage of picture/layer raster cache in DevTools

### ISSUE THREAD

**Issue by liyuqian** [flutter/flutter#56719]
Internal: b/160377936

**Comment by dnfield** [comment]
This could probably be handled similarly to some of the API in https://github.com/flutter/flutter/compare/master...devoncarew:image_cache_apis

**Comment by zanderso** [comment]
@liyuqian I see you bumped this up to P2. P2 issues should specify who the customer is with a customer tag. Could you please add one, or drop the priority back down? Thanks!

**Comment by liyuqian** [comment]
Yes, that's correct. Thanks!

**Comment by Hixie** [comment]
@liyuqian says that this is on deck to be dealt with after #53607 which he is currently working on.

**Comment by chinmaygarde** [comment]
Triage: [53607](https://github.com/flutter/flutter/issues/53607) has been closed so @liyuqian should be unblocked on working on this.

**Comment by liyuqian** [comment]
Update: the work on the engine side is done and we're now waiting for Terry to pick up the work on the DevTools side after his vacation.

**Comment by zanderso** [comment]
Is there a separate issue for the DevTools work? Should we close this issue if the work described in here is done?

**Comment by liyuqian** [comment]
Currently there's no separate issue for DevTools. I'm updating this issue's title to clarify that we need to expose the memory usage all the way through DevTools. I also scheduled a meeting with Terry next week when he's back to discuss more details. If the work on DevTools side is significant, we can open a new issue.

**Comment by Hixie** [comment]
(Downgrading to P3 after talking with @mehmetf)

**Comment by terrylucas** [comment]
Code is written to call the service extension.  Pinged liyuqian, as I'm getting an error that '_flutter.estimateRasterCacheMemory' service is not known.  Using tip of Flutter:
```
Flutter 1.22.0-10.0.pre.205 • channel master • https://github.com/flutter/flutter.git
Framework • revision 0220c3585c (3 hours ago) • 2020-09-16 23:13:05 +0800
```

**Comment by terrylucas** [comment]
Fixed with [PR](https://github.com/flutter/devtools/commit/709fa720442af8d219dee391729c5bc4b6ea41a8).
